src/Airship/Internal/Decision.hs

```diff
@@ -95,12 +95,9 @@ newtype IfNoneMatch = IfNoneMatch ByteString
 -- Decision Helpers
 ------------------------------------------------------------------------------
 
-negotiateContentTypesAccepted :: Monad m => Resource m -> FlowStateT m ()
-negotiateContentTypesAccepted Resource{..} = do
+negotiateContentTypesAccepted :: Monad m => [(MediaType, Webmachine m a)] -> FlowStateT m a
+negotiateContentTypesAccepted accepted = do
```

Comment:
Have tweaked this to make it more re-reuable. I'm about 99.99% sure there isn't a code path that is now passing in `contentTypesAccepted` that was previously using `patchContentTypesAccepted`. Will need to be careful when `o14` or similar is wired in.